DeAceroDLabs/marketplace-frontend
# [DMP-682] Update card sizes on Home

## What is done

[[DMP-682] Actualizar tamaño de tarjetas en Home](https://deacero.atlassian.net/browse/DMP-682)

## How is done

1. Use cards of size medium instead of small on Home.
2. Display 4 items on each row.

## How to test

1. Run the project via `npm start`
4. On home page you will see 4 items per row on the products section.
5. The loading state will display 4 cards.
6. Once the products are loaded you will see a "ver más" button at the bottom if the products length is greater than 8.

## Evidence
Before

<img width="1267" alt="Screen Shot 2022-11-01 at 11 36 27" src="https://user-images.githubusercontent.com/68872548/199311699-d86df01c-cd11-4990-b497-09f2fa861caf.png">

After
<img width="1400" alt="Screen Shot 2022-11-01 at 11 35 48" src="https://user-images.githubusercontent.com/68872548/199311753-8b24080b-511c-4836-86d0-fa374ff3dede.png">
